packages/styles/src/card.scss

```diff
@@ -323,6 +322,10 @@ $fd-card-header-outline-offset: 0.0625rem !default;
       overflow: hidden;
     }
 
+    .#{$list} {
+      @include fd-set-paddings-x(0.125rem);
+    }
```

Comment:
I don't think this is correct! The card content should have additional 1rem left and right paddings:
![Screenshot 2023-10-31 at 12 21 28 PM](https://github.com/SAP/fundamental-styles/assets/39598672/8315cc8b-ec2f-4922-942f-75895662ce19)

Reply:
The card content does have 1rem of padding, and I think the figma note is saying there should be 4px more of padding for the list, so I think I need to bump this up to 0.25rem

Reply:
![Screenshot 2023-10-31 at 3 04 10 PM](https://github.com/SAP/fundamental-styles/assets/39598672/b3034f2b-9251-45fd-be64-f523870a9d81)

The card content by default doesn't have paddings. It's left to the applications to add, if needed. In this case 1rem left and right paddings should be applied.